/video_compressor:
  ros__parameters:
    video0:
      input_topic: img_raw
      output_topic: img_viewable
      desired_width: 480
    video1:
      input_topic: img/red
      output_topic: image_red_viewable
      desired_width: 100
    video2:
      input_topic: img/green
      output_topic: image_green_viewable
      desired_width: 100
    video3:
      input_topic: img/blue
      output_topic: image_blue_viewable
      desired_width: 100
